keystore/src/entities/platform/wasm/mls/epoch_encryption_keypair.rs

```diff
@@ -67,14 +67,14 @@ impl Entity for MlsEpochEncryptionKeyPair {
     }
 
     fn encrypt(&mut self, cipher: &aes_gcm::Aes256Gcm) -> CryptoKeystoreResult<()> {
-        self.keypairs = Self::encrypt_data(cipher, self.keypairs.as_slice(), self.aad())?;
+        self.keypairs = Self::encrypt_data(cipher, self.keypairs.as_slice(), &self.aad()?)?;
         Self::ConnectionType::check_buffer_size(self.keypairs.len())?;
 
         Ok(())
     }
 
     fn decrypt(&mut self, cipher: &aes_gcm::Aes256Gcm) -> CryptoKeystoreResult<()> {
-        self.keypairs = Self::decrypt_data(cipher, self.keypairs.as_slice(), self.aad())?;
+        self.keypairs = Self::decrypt_data(cipher, self.keypairs.as_slice(), &self.aad()?)?;
```

Comment:
I'm wondering, if all the entities use the very same AAD, can we pull the `aad()` call out of entities and into `EntityBase`?

Reply:
Not sure what you mean? `aad()` is a method on `Entity` (which is a trait defined for SQLite and wasm seprately, and `aad()` is needed only on wasm).

Reply:
Since `decrypt_data` is always called with the result of a call to `instance.aad()`, I thought it'd be more robust to just make that call inside of `Entity::decrypt_data` instead of the callers.

Reply:
But feel free to ignore this, as it's orthogonal to the topic of this PR.